docs/site/getting-started/getting-started-shopping-cart-app.md

```diff
@@ -151,7 +151,7 @@ For the `shopping cart` micro application we need to implement an activator to r
 
 ***
 #### What is an Activator
-An activator allows a micro application to initialize and connect to the platform when the user loads the host application into his browser. In the broadest sense, an activator is a kind of microfrontend, i.e. an HTML page that runs in an iframe. In contrast to regular microfrontends, however, at platform startup, the platform loads activator microfrontends into hidden iframes for the entire platform lifecycle, thus, providing a stateful session to the micro application on the client-side.
+An activator allows a micro application to initialize and connect to the platform upon host application's startup, i.e., when the user loads the web application into his/her browser. In the broadest sense, an activator is a kind of microfrontend, i.e. an HTML page that runs in an iframe. In contrast to regular microfrontends, however, at platform startup, the platform loads activator microfrontends into hidden iframes for the entire platform lifecycle, thus, providing a stateful session to the micro application on the client-side.
```

Comment:
Instead of `his/her browser`, we could write `the browser` to be more neutral.
There seems to be a line-ending issue as well, because the last dot is marked red.

Reply:
I checked the line endings, they are all "LF" just as we want them. Strange that we have a change here.